Plugins/Skinning/src/SkinningComponent.cpp

```diff
@@ -129,6 +144,28 @@ void SkinningComponent::initialize() {
 
         m_isReady = true;
         setupSkinningType( m_skinningType );
+
+        // prepare RO for skinning weights display
+        auto ro = getRoMgr()->getRenderObject( *m_renderObjectReader() );
+        m_baseTechnique = ro->getRenderTechnique();
+        m_baseUV = ro->getMesh()->getData( Ra::Engine::Mesh::VERTEX_TEXCOORD );
+        m_weightTechnique.reset( new Ra::Engine::RenderTechnique );
+        auto builder = Ra::Engine::EngineRenderTechniques::getDefaultTechnique( "BlinnPhong" );
+        builder.second( *m_weightTechnique.get(), true );
+        auto matT = std::static_pointer_cast<Ra::Engine::BlinnPhongMaterial>(
+            ro->getRenderTechnique()->getMaterial() );
+        std::shared_ptr<Ra::Engine::BlinnPhongMaterial> mat(
+            new Ra::Engine::BlinnPhongMaterial( "IS2016_Mat" ) );
+        mat->m_kd = Ra::Core::Utils::Color::Skin();
+        mat->m_ks = Ra::Core::Utils::Color::White();
+        m_weightTechnique->setMaterial( mat );
+        // assign texture
+        Ra::Engine::TextureParameters texParam;
+        texParam.name = "../../../Assets/Textures/Influence0.png";
```

Comment:
This path is wrong.
If you want to use assets they need to be installed alongside the application so the path as runtime is known and guaranteed to be correct.

Reply:
Also, the mentioned file is not part of this commit, and not in the trunk....

Reply:
The resources issue is already known , see #337 .
The mentionned file is in Assets/Textures since February 26, 2016.

Reply:
What I am asking is to copy/link the file in a subdirectory of `bin`, like we do for the shaders.
I cannot accept we add a path like that in the code.